core/js/src/main/scala/zio/internal/PlatformSpecific.scala

```diff
@@ -35,6 +35,13 @@ private[internal] trait PlatformSpecific {
     val _ = action
   }
 
+  /**
+   * Adds a signal handler that executes the specified action on receiving a signal.
+   */
+  @silent("never used")
+  def addSignalHandler(signal: String)(handler: () => Unit): Boolean =
+    true
```

Comment:
Should this actually return `false`? I thought the meaning of the return value was `true` if the signal handler was successfully installed and `false` otherwise. Here the signal handler was not successfully installed (admittedly because we don't currently support installing signal handlers on ScalaJS at all). So I think `false` is most accurate in communicating to the caller that the signal handler they tried to install is never going to run.